### PR TITLE
chore:add volunteer itzrahulyadav

### PR DESCRIPTION
**Pull request by itzrahulyadav**

### CODE REVIEW

File: data/users/itzrahulyadav.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Rahul Yadav",
+    "bio": "Web developer",
+    "image": "https://avatars.githubusercontent.com/u/65400893?s=96&v=4",
+    "categories": [
+      {
+        "slug": "cloud",
```
Comment (codewithvoid) [inline data/users/itzrahulyadav.json:7]: use devops slug here

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @itzrahulyadav. Added small comment.

**Comment by itzrahulyadav** [comment]
Done.Thanks for the suggestion.

**Review by codewithvoid — APPROVED** [review]
Thanks. Welcome!

**Pull request merged**